docs/troubleshoot/troubleshooting.md

```diff
@@ -2,6 +2,29 @@
 
 To isolate and resolve Zowe&trade; problems, you can use the troubleshooting and support information.
 
+## How to start troubleshooting
+
+When you run into some issues and look for troubleshooting tips, the following steps may help you.
+
+1. Search the error message or error code that you have in log on [Zowe doc site](https://docs.zowe.org/). If there are existing solutions, then you could follow them.
+
+2. If no solution is available or the existing solutions cannot apply to your problems, you could search the keywords, error messages, or error code in [Zowe GitHub repository](https://github.com/zowe). If the issues or pull requests are closed with solutions, you could follow them. If the issues are still open, you could leave your question or have a discussion in the thread to solve your issue.
```

Comment:
missing a "the": "... or error code in the Zowe GitHub repository." I'd also make "problems" singular.

Also suggesting: If you find a closed issue or pull request, try troubleshooting by using the information shared in the item's Conversation section. If an issue is still open, post a question or comment to prompt a discussion on your problem.